plugin_tests/client/histomicstkSpec.js

```diff
@@ -0,0 +1,78 @@
+/* globals girder, girderTest, describe, it, expect, waitsFor, runs */
+
+girderTest.addScripts([
+    '/clients/web/static/built/plugins/jobs/plugin.min.js',
+    '/clients/web/static/built/plugins/worker/plugin.min.js',
+    '/clients/web/static/built/plugins/large_image/plugin.min.js',
+    '/clients/web/static/built/plugins/slicer_cli_web/plugin.min.js',
+    '/clients/web/static/built/plugins/HistomicsTK/plugin.min.js'
+]);
+
+girderTest.startApp();
+
+describe('Test the HistomicsTK plugin', function () {
+    it('change the HistomicsTK settings', function () {
+        girderTest.login('admin', 'Admin', 'Admin', 'password')();
+        waitsFor(function () {
+            return $('a.g-nav-link[g-target="admin"]').length > 0;
+        }, 'admin console link to load');
+        runs(function () {
+            $('a.g-nav-link[g-target="admin"]').click();
+        });
+        waitsFor(function () {
+            return $('.g-plugins-config').length > 0;
+        }, 'the admin console to load');
+        runs(function () {
+            $('.g-plugins-config').click();
+        });
+        waitsFor(function () {
+            return $('input.g-plugin-switch[key="HistomicsTK"]').length > 0;
+        }, 'the plugins page to load');
+        girderTest.waitForLoad();
+        runs(function () {
+            expect($('.g-plugin-config-link[g-route="plugins/HistomicsTK/config"]').length > 0);
+            $('.g-plugin-config-link[g-route="plugins/HistomicsTK/config"]').click();
+        });
+        waitsFor(function () {
+            return $('#g-histomicstk-form input').length > 0;
+        }, 'settings to be shown');
+        girderTest.waitForLoad();
+        var styles = [{'lineWidth': 8, 'id': 'Sample Group'}];
+        var styleJSON = JSON.stringify(styles);
```

Comment:
It's probably best to keep definitions at the top of functions given how the individual steps run async.